tap_s3_csv/__init__.py

```diff
@@ -94,7 +95,8 @@ def do_sync(config, catalog, state):
     
     # Exports logs for row and col count
     json_row_col = {"name": name, "row": total_row_count, "col": total_col_count}
-    LOGGER.info("EXPORTS tap-s3-csv data_props: " + str(json_row_col))
+    grouped_logs.insert(0,"EXPORTS tap-s3-csv data_props: " + str(json_row_col) + "\n")
+    LOGGER.info("|".join(grouped_logs))
```

Comment:
guess we no longer need the `|` if we already append new row for each line of log?

Reply:
one question for the `total_col_count`, say we have two sheets one with 10 columns, one with 20 columns, does the total_col_count just do 10+20?

Reply:
Yes we would have that for quick checks. We would also have the individual ones if we want a more accurate look.

Reply:
I can change it to use max columns if you think it would be better

Reply:
that should be fine, just fear this `total_col_count` value could be miss leading.

Reply:
Also need the | for script to easier separate the object that we want in it